scripts/rhoam-manifest-generator.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# Set the service name and version
+SERVICE_NAME="services-rhoam"
+
+RHOAM_VERSION=$(grep managed-api-service.v deploy/olm-catalog/managed-api-service/managed-api-service.package.yaml | tail -c 6)
+
+# Manifest folder search
+MW_MANIFESTS=( 3scale cloud-resources grafana marin3r rhsso )
+
+# Pre-filetered manifest file
+pre_sorted_file="pre-sorted-file.txt"
+
+# Output file name
+file="${RHOAM_VERSION}-manifest.txt"
+
+# Dependencies used
+go mod graph | cut -d " " -f 2 | tr @ - | while read x; do echo "${SERVICE_NAME}:${RHOAM_VERSION}/$x" >> "rhoam-manifests/$pre_sorted_file"; done
+
+# Remove repeating dependencies
+sort -u "rhoam-manifests/$pre_sorted_file" >> "rhoam-manifests/$file"
+
+# Delete pre-sorted file
+rm -rf "rhoam-manifests/$pre_sorted_file"
```

Comment:
```suggestion
rm -f "rhoam-manifests/$pre_sorted_file"
```
if the `--recursive` flag is required then I feel you `pre_sorted_file` variable needs to be renamed.